src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -64,26 +64,33 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = CollectAlignmentSummaryMetrics.USAGE_SUMMARY + CollectAlignmentSummaryMetrics.USAGE_DETAILS,
-        usageShort = CollectAlignmentSummaryMetrics.USAGE_SUMMARY,
+        usage = "Using read outputs from NGS technologies, this tool provides detailed metrics" +
+        "regarding the quality of the alignments of these reads to a reference sequence. These metrics include " +
+        "the numbers of " +
+        "reads that passed machine signal-to-noise threshold quality filters as well as values representing the " +
+        "degree of alignment to a reference sequence.  Please see http://broadinstitute.github.io/picard/picard-" +
+        "metric-definitions.html#AlignmentSummaryMetrics for detailed explanations of each metric.  Additional " +
+        "information can be found by downloading the following documents: (http://support.illumina.com/content/" +
```

Comment:
Should make it clear that the machine quality filter metrics stuff refers specifically to Illumina technology.